Makefile

```diff
@@ -120,3 +122,13 @@ docker-shell:
 check:
 	clear
 	@docker-compose run --rm web python manage.py check
+
+make-messages:
+	clear
+	@docker-compose run --rm web python manage.py makemessages -l en
+	@docker-compose run --rm web python manage.py makemessages -l af
+
+compile-messages:
+	clear
+	@docker-compose run --rm web python manage.py compilemessages -l en
+	@docker-compose run --rm web python manage.py compilemessages -l af
```

Comment:
Same here. According to `compilemssages --help`  compilemessages processes all files by default. Maybe good enough for us?

Reply:
Okay, will try this. Thanks